contracts/src/v0.8/dev/keeper2_0/KeeperRegistryBase2_0.sol

```diff
@@ -373,11 +374,11 @@ abstract contract KeeperRegistryBase2_0 is ConfirmedOwner, ExecutionPrevention {
     // Divide l1CostWei among all batched upkeeps. Spare change from division is not charged
     l1CostWei = l1CostWei / numBatchedUpkeeps;
 
-    uint256 gasPayment = ((weiForGas + l1CostWei) * 1e18) / linkNative;
-    uint256 premium = (gasPayment * hotVars.paymentPremiumPPB) / 1e9 + uint256(hotVars.flatFeeMicroLink) * 1e12;
+    gasPayment = uint96(((weiForGas + l1CostWei) * 1e18) / linkNative);
```

Comment:
I prefer to have type conversion after the `if (gasPayment + premium > LINK_TOTAL_SUPPLY) revert PaymentGreaterThanAllLINK();` check which would revert instead of silently typecasting large values